src/igraph/drawing/graph.py

```diff
@@ -1180,7 +1180,8 @@ def callback_edge_offset(event):
             if default is True:
                 default = 0.5
             default = float(default)
-            kwds["edge_curved"] = autocurve(graph, attribute=None, default=default)
+            autocurve(graph, attribute="curved", default=default)
+            kwds["edge_curved"] = graph.es.get_attribute_values("curved")
```

Comment:
Btw. Not sure if this kwds is used in the code. Autocurve returns none --> prior code pushed None in this key. Now it contained the actual values.

Reply:
`autocurve()` is supposed to return `None` if and only if `attribute` is not `None`, otherwise it should return the calculated curvatures, so I'm not 100% sure whether this should be fixed here or in `autocurve()` itself. I'll investigate a bit.

Reply:
`autocurve()` seems to return a proper list here so this part of the patch is not needed; the real problem is that `kwds["edge_curved"]` is not used later. This part was copied from the `DefaultGraphDrawer` where `kwds` _is_ used later so that's the part that needs fixing.